### DISCUSSION TITLE
dials.export to PETS2

### DISCUSSION THREAD

**Discussion by Javalde91** [dials/dials#2637] [General]
Hello everyone,

I'm currently working on exporting data for Jana2020 refinements after merging datasets on cosym. However, I'm encountering some difficulties in understanding the process based on the information available on the webpage.

Here's the code line I've been using:

dials.export scaled.expt scaled.refl format=pets

I'm uncertain about what to add next for dials_dyn.cif_pets.

Additionally, I'm unsure whether it's possible to perform the export directly on cosym. I'm also uncertain whether using scaled.expt or scaled.refl is correct, or if I should use integrated instead.

Any assistance or clarification would be greatly appreciated.

Thank you.

**Comment by dagewa** [comment]
Hi, PETS export is for individual `integrated.{expt,refl}` data sets only. That's exactly how PETS also does it - I don't believe PETS does any data scaling, but some scaling might be done in Jana2020 (I'm not sure). I don't think PETS export should be used for data sets combined by `dials.cosym`, or data sets scaled by `dials.scale`.

**Reply by dagewa** [reply]
To add more details: the dynamic diffraction refinement done in Jana2020 needs detailed information about the orientation of the crystal at every frame. This is why it can't be done with data sets combined across multiple crystals, which are randomly oriented.
